pkg/kubelet/kuberuntime/kuberuntime_gc.go

```diff
@@ -37,6 +37,7 @@ type containerGC struct {
 	client           internalapi.RuntimeService
 	manager          *kubeGenericRuntimeManager
 	podStateProvider podStateProvider
+	symlinkFirstSeen map[string]time.Time
```

Comment:
symlinkFirstSeen is actually when the symlink is first 'not' seen, perhaps a name change for this variable?

Reply:
How does symlinkFirstObservedMissing sound ?